packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -16,9 +19,7 @@ const propTypes = {
   /**
    * An array of objects to be displayed as quick link options.
    */
-  quickLinks: PropTypes.arrayOf(PropTypes.shape({
-    text: PropTypes.string.isRequired,
-  })),
```

Comment:
why was this removed?

Reply:
right, makes sense, will revert this change